assets/stylesheets/components/_media-objects.scss

```diff
@@ -0,0 +1,172 @@
+/**
+** Media Objects
+** 1. Base layout
+** 2. Text
+** 3. Images
+** 4. Overrides
+** 5. Connect-specific layout
+**/
+
+
+
+
+// Base layout
+.media {
+  border-top: 1px solid rgba(darken($cr-gray-lighter, 10), .4);
+  cursor: pointer;
+  display: flex;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+
+  > div:last-child {
+    margin-left: 1rem;
+  }
+
+  .media-body {
+    flex-grow: 1;
+    width: initial;
+  }
+}
+
+
+
+
+// Text
+.media-heading {
+  @extend .component-header;
+  color: $cr-gray-dark;
+  margin-bottom: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  &.media-heading-link {
+    color: $cr-cyan;
+  }
+}
+
+.say-hi {
```

Comment:
I'm not sure what the name of this should be tbh. It's not a button though - it's a line of text that sometimes shows up over the host/me labels or participant's name. The text says "Say Hi to". This could probably be split out into the `connect-specific layout` section.